app.test.js

```diff
@@ -74,4 +74,32 @@ beforeEach(() => {
     });
   });
   });
+  describe('/api/reviews', () => {
+    it('GET 200: should respond with a reviews array of review objects ', () => {
+      return request(app)
+      .get('/api/reviews')
+      .expect(200)
+      .then(({body}) => {
+        const {reviews} = body
+        expect(reviews).toBeInstanceOf(Array);
```

Comment:
i think its just missing one expect/test - you can do it inside this test if you want

 Reviews need to be sorted by descending date. jest-sorted is recommended  for testing this.